app/views/errors/not_found.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_for :breadcrumbs do %>
+    <%= home_breadcrumb %>
+<% end %>
```

Comment:
I think that all the error pages have the same layout. But we are repeating it always. Can we extract a layout file here?